src/Feldspar/Compiler/Imperative/FromCore.hs

```diff
@@ -123,19 +123,19 @@ fromCoreUT opt funname uast = (Module defs, maxVar')
     (outParam,maxVar',results) = runRWS (compileProgTop uast) (initEnv opt) maxVar
 
     decls      = decl results
```

Comment:
The long block comment about fast returns should be updated to reflect the change.

Reply:
Missed that, Will do!

Reply:
The comment is still valid as it stands. There is still the distinction between fast returns and, um, slow returns. It's just the representation (of both types of return) in the data type that is changed.